.github/workflows/labeler.yml

```diff
@@ -7,6 +7,8 @@ on:
       - reopened
       - closed
 
+permissions:
+  contents: read
```

Comment:
This is handled in jobs. Why do we need it again here?

Reply:
Right , I initially didn't pay attention, but as I mentioned in the pr message , below is what ossf scorecard say about reward points after security checks.

<img width="652" alt="image" src="https://github.com/WasmEdge/WasmEdge/assets/92215138/61242fdf-d4f7-4d29-a458-b7ce7519eaff">

other than that I'll see if some of them redundant and remove them.

Reply:
Also , projects with highest ossf score have followed this